packages/react/src/tokens/themes/breakpoint.constants.ts

```diff
@@ -0,0 +1,460 @@
+import { FontWeight } from './theme.constants';
+
+const fontFamily = 'neue-haas-grotesk-display';
+
+export const smallDesktop = {
```

Comment:
Could we only have one object with all theme?
The only thing that changes are the values, so if we calculate each one for each breakpoint it will be more easy.